README.md

```diff
@@ -142,6 +142,44 @@ or
 
 and select the profiles you want to build.
 
+### Building Eclipse Kura Docker/Podman Containers
+
+The kura container build process currently only supports x86 containers. Following the instructions below will build two containers. One based on Alpine Linux ```kura-alpine-x86_64```, and another on Ubi8 ```kura-ubi8-x86_64```.
+
+Change to the new directory and clone the Kura repo:
+
+```bash
+git clone -b develop https://github.com/eclipse/kura.git
+```
+
+Move inside the newly created directory and build the target platform:
+
+```bash
+mvn -f target-platform/pom.xml clean install
+```
+
+Build the core components:
+
+```bash
+mvn -f kura/pom.xml clean install
+```
+
+Build the examples (optional):
+
+```bash
+mvn -f kura/examples/pom.xml clean install
+```
+
+Build the target profiles:
+
```

Comment:
This instructions are already explained [here](https://github.com/eclipse/kura#build-kura). There's no need to repeat ourselves. Just reference them.

```suggestion
Build Kura as per [our instructions](https://github.com/eclipse/kura#build-kura). To build the containers you'll need to change the target of the "Build the target profiles" step like the following:

```